docker-vertica/Dockerfile

```diff
@@ -87,66 +87,72 @@ RUN set -x \
   && chmod 777 -R /opt/vertica
 
 ##############################################################################################
-FROM centos:centos${OS_VERSION}
+FROM ubuntu:${BASE_OS_VERSION}
 
 ARG DBADMIN_GID=5000
 ARG DBADMIN_UID=5000
 
 COPY --from=builder /opt/vertica /opt/vertica
 COPY --from=builder /home/dbadmin /home/dbadmin
 COPY --from=builder /root/.ssh /root/.ssh
-COPY --from=builder /etc/ssh /etc/ssh
 COPY --from=builder /var/spool/cron /var/spool/cron
 
 ENV LANG en_US.utf8
 ENV TZ UTC
 ENV PATH "$PATH:/opt/vertica/bin:/opt/vertica/sbin"
+ENV DEBIAN_FRONTEND noninteractive
 
 ADD ./packages/init.d.functions /etc/rc.d/init.d/functions
 
+
+# make the "en_US.UTF-8" locale so vertica will be utf-8 enabled by default
+RUN apt-get update && apt-get install -y locales && rm -rf /var/lib/apt/lists/* \
+	&& localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
+
+
 RUN set -x \
   # COPY may not preserve file permissions on older docker versions.  This is a
   # workaround for that.  This will cause the image to increase a bit, but there
   # isn't much we copy over for this directory.
   && chown -R $DBADMIN_UID:$DBADMIN_GID /home/dbadmin \
   # Update is needed to be confident that we're picking up
   # fixed libraries.  We depend on malware check of container afterwards 
-  && yum -q -y update \
-  && yum -y update --security \
-  # CentOS 8 - enable powertools to fix locales issue
-  && bash -c "if [ \"$(rpm -E %{rhel})\" == '8' ]; then yum install -q -y dnf-plugins-core glibc-locale-source; yum -q config-manager --set-enabled powertools; fi" \
-  && yum install -y \
-  cronie \
+  && apt-get -y update \
+  && apt-get install -y \
+  cron \
+  sysstat \
+  ca-certificates \
+  ntp \
   dialog \
-  iproute \
-  krb5-workstation \
-  mcelog \
+  iproute2 \
+  libkeyutils1\
+  procps \
+  krb5-user \
   logrotate \
+  openssh-client \
   openssh-server \
-  openssh-clients \
   openssl \
   sudo \
-  which \
-  # CentOS 8 - fixes - Unsupported locale character encoding: use a utf8 locale, not a ANSI_X3.4-1968 locale
-  && bash -c "if [ \"$(rpm -E %{rhel})\" == '8' ]; then localedef -i en_US -f UTF-8 en_US.UTF-8; fi" \
+  && ssh-keygen -q -A \
   && /usr/sbin/groupadd -r verticadba --gid ${DBADMIN_GID} \
   && /usr/sbin/useradd -r -m -s /bin/bash -g verticadba --uid ${DBADMIN_UID} dbadmin \
+  && /etc/init.d/ssh start \
   # Allow passwordless sudo access from dbadmin
   && echo "dbadmin ALL=(ALL) NOPASSWD: ALL" | tee -a /etc/sudoers \
-  && yum clean all \
-  && /bin/rm -rf /var/cache/yum \
-  # Used by the agent to see if networking is enabled.  Empty file is okay
-  && touch /etc/sysconfig/network \
+  && echo "dbadmin -       nofile  65536" >> /etc/security/limits.conf \
   # Permit ssh connections
-  && rm -rf /run/nologin
-
+  && rm -rf /run/nologin \
+  # Moving this because on Ubuntu crontab files are in /var/spool/cron/crontabs/
+  && mv /var/spool/cron/dbadmin /var/spool/cron/crontabs/ \
```

Comment:
We copy dbadmin at line 98.  Can we just change the destination directory and remove this line?